### DISCUSSION TITLE
:sob: looks like something not installed properly.

### DISCUSSION THREAD

**Discussion by ruru-m07** [end-4/dots-hyprland#968] [Q&A]
this what my window looks like. i have no idea what's going on

![image](https://github.com/user-attachments/assets/b1a719da-9988-43ea-9a6e-fb4c9788431c)

![image](https://github.com/user-attachments/assets/6afa3746-38a2-4dfe-8985-eb0c22c46f81)

i'm new with linux :(

**Comment by prathamreet** [comment]
take required data backup and reinstall windows, try these expreiments in virtual machine instead on local machine !!

**Reply by ruru-m07** [reply]
:sob: i just add a wallpaper and it fixed. IDK why but it fix :) 

( now i have new issue. i can't able to stream )
